README.md

```diff
@@ -100,6 +100,18 @@ To see the full list of management commands use `help`.
 docker-compose run --rm manage help
 ```
 
+### Import Postman collection
+Postman is an interactive tool for verifying the APIs of your project. We've created a shared Postman collection (a .json file) in the `postman` folder to help contributors more easily reproduce observed behaviour in our dev API. To get it set up, please follow these steps:
```

Comment:
I think having a link or two with resources to learn about the how and why behind verifying APIs on a project would be good.

Reply:
How about: 
```
Postman is a free interactive tool for verifying the APIs of your project. You can download it at postman.com/downloads. 

We've created a shared Postman collection (a .json file) in the `postman` folder to help contributors more easily reproduce observed behaviour in our dev API, which is necessary because our frontend app at github.com/codebuddies/frontend will make requests to this API. To get it set up, please follow these steps:```

Reply:
(Feel free to make edits to that btw!)

Reply:
Downloading Postman
Please make sure it is at least v7.6.0, if installed, or you are downloading the latest stable version.
Linux,
- Distro package manager:
- use the search feature to find in your package manager
- (RECOMMENDED) Flatpak
- After setting up flatpak it through flatpak using flatpak install postman and enter "yes"/"y" for all the questions it will ask. Flatpak is designed to provide the most up-to-date versions of software for most distros, so if you have the option, use Flatpak to guarantee Linux OS compatibility and to keep Postman up-to-date.